client/coral-embed-stream/src/CommentStream.js

```diff
@@ -272,8 +274,8 @@ const mapDispatchToProps = (dispatch) => ({
   getStream: (rootId) => dispatch(getStream(rootId)),
   addNotification: (type, text) => dispatch(addNotification(type, text)),
   clearNotification: () => dispatch(clearNotification()),
+  postAction: (item, action, itemType, field, detail) => dispatch(postAction(item, action, itemType, field, detail)),
```

Comment:
I'm using onClickOutside or something like that in Permalink if you want to check it out @davidgljay